use-cases/single-file-C-program/test-main.c

```diff
@@ -33,33 +35,41 @@ timespec_to_ns(struct timespec *ts)
 }
 
 /**
- * The sequence of l3_log_fast() and l3_log_simple() is flipped in this version
- * as opposed to the .cpp and .cc versions of this test exerciser program.
+ * The sequence of l3_logging-APIs is flipped in this version of the sample
+ * program, as opposed to the .cpp and .cc versions of this exerciser program.
  * This is done so that we can see the different 'arg1' values stashed away
- * by the l3_log_simple() call, when unpacked by the l3_dump.py script.
+ * by the slow-logging() call, when unpacked by the l3_dump.py script.
```

Comment:
Removed all references to the L3-logging APIs from comments as it was cofusing the grep / parsing in the helper pytest method, `build_exp_loc_msg_list()`.